hack/build/push-image.sh

```diff
@@ -14,11 +14,14 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-set -e
+set -eux -o pipefail
 
 PORT=${1}
 
 script_dir="$(cd "$(dirname "$0")" && pwd -P)"
 source "${script_dir}"/../config.sh
 
-buildah push --tls-verify=false ${IMAGE}:${VERSION} docker://${DOCKER_PREFIX}/${IMAGE_NAME}:${VERSION}
\ No newline at end of file
+# check why this buildah does not see a login,
+echo "docker_prefix: $DOCKER_PREFIX, docker_tag: $DOCKER_TAG"
+
+buildah push --tls-verify=false ${IMAGE}:${VERSION} docker://${DOCKER_PREFIX}/${IMAGE_NAME}:${VERSION}
```

Comment:
Never use --tls-verify=false what is the reason we are doing that?

Reply:
There must be some misconfiguration with local registry. Either we are asking on https and it runs on http or the other way around.

Reply:
What is the difference between DOCKER_TAG and VERSION? on this line VERSION appears to be the tag.

If you want to push to the kubevirtci local registry you do the following steps (you can verify this manually). After looking up the port you need:
```bash
buildah bud -t localhost:${port}/kubevirt/velero-plugin:${tag}
```
with tag being the version. This should put an image called localhost:port/kubevirt/velero-plugin:tag in your local registry

Then you can push with:
```bash
buildah push localhost:${port}/kubevirt/velero-plugin:${tag}
```

If you have a kubevirtci cluster running. It will then get pushed directly to the registry in that cluster. You can then reference your image in that cluster with registry:5000/kubevirt/velero-plugin:${tag}

Reply:
So your DOCKER_PREFIX should be localhost:port for testing and quay.io for actual pushing.

Reply:
yes, that is a mess, and I have fixed most of that yesterday, will push shortly.

Reply:
but I need to check what can be done with local cluster registry, so it do not need --tls-verify=false. Maybe you have some config in hpp operator